src/style.css

```diff
@@ -1,3 +1,75 @@
-body {
+:root {
+  --blue: #0070f3;
+}
+
+* {
   margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: Roboto, sans-serif, bold;
+}
+
+header {
```

Comment:
Don't use tag names for styling (except html and body)
Fix it in all code.

![image](https://github.com/mate-academy/layout_moyo-header/assets/54860641/91feef38-ac6a-4612-b6c3-f87fc96ae7f4)